src/components/productView/ProductView.module.css

```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Cantata+One&family=Montserrat:wght@400;600;700&display=swap");
+/******* Product custom css start *******/
+* {
+  font-family: "Montserrat";
+}
+.Img1 {
```

Comment:
why did you write class names in PascalCase? They should be camelCase.

Reply:
className casing problem fixed